doc/EPIC.md

```diff
@@ -0,0 +1,110 @@
+This is especially important for hidden paths [[2]](#2). Hidden paths are paths which are not publicly announced, but only communicated to a group of authorized sources. If one of those sources sends traffic on the hidden path using SCION path type packets, an adversary can observe the MACs and reuse them to send traffic on the hidden path himself. This allows the adversary to reach services that were meant to be hidden, or to launch DOS-attacks directed towards them.
+EPIC precludes such attacks, making hidden paths more secure.
+
+## EPIC-HP Overview
+EPIC-HP (EPIC for Hidden Paths) provides the improved security properties of EPIC on the very last inter-AS link of a path. It is meant as a lightweight EPIC version and is specifically designed to better protect hidden paths.
+
+### Assumptions
+EPIC-HP makes the following assumptions necessary to provide a meaningful level of security:
+- The AS protected by the hidden path is the last AS.
+By "last AS" we mean that the beacon which defined the hidden path ends in the local interface of this AS. Or stated differently: the AS behind the hidden path does not forward the beacon defining the hidden path to further downstream/peering ASes. 
+- On the interface-pairs (ingress/egress pair) that affect the hidden path, the last two ASes either:
+  1. Only allow EPIC-HP path type traffic. See use case "Highly Secure Hidden Paths" [here](#HighlySecureHiddenPaths). The path type filtering is further explained [here](#PathTypeFiltering).
+  2. Prioritize EPIC-HP path type traffic. See use case "DOS-Secure Hidden Paths" [here](#DOSSecureHiddenPaths).
+- The last two ASes of the hidden path have a duplicate-suppression system in place. This prohibits DOS attacks based on replayed packets.
+
+### Example
+The following figure illustrates those assumptions:
+<p align="center">
+  <img src="fig/EPIC/path-type-filtering.png" width="630">
+</p>
+
+Here, AS 6 is the AS protected by the hidden path (blue lines). The hidden path ends at the local interface (black dot) of AS 6, so AS 6 did not forward the beacon that defines the hidden path further down to AS 7. 
+This is however still allowed for SCION path type traffic (green lines): there are SCION paths that enter AS 6 from AS 4. One of the two paths ends in the local interface of AS 6, while the other one is extended further to AS 7. 
+
+In this example, the border routers of AS 6 and AS 5 (the last and penultimate ASes on the hidden path) further implement path type filtering (red dots). Metaphorically speaking , they do not allow green lines to overlap with blue lines. For example, AS 5 will block (red "X" in the figure) SCION path type traffic from AS 3 that is destined towards AS 6, as it would affect the hidden path. Instead of blocking non-EPIC-HP path type traffic, ASes could also prioritize EPIC-HP traffic, which would still satisfy the assumptions above.
```

Comment:
```suggestion
In this example, the border routers of AS 6 and AS 5 (the last and penultimate ASes on the hidden path) further implement path type filtering (red dots). For example, AS 5 will block (red "X" in the figure) SCION path type traffic from AS 3 that is destined towards AS 6, as it would affect the hidden path. Instead of blocking non-EPIC-HP path type traffic, ASes could also prioritize EPIC-HP traffic, which would still satisfy the assumptions above.
```